frontend/src/bundles/overview/components/circle-progress/circle-progress.tsx

```diff
@@ -33,16 +33,16 @@ const CircleProgress = ({
 }: CircularProgressProperties): JSX.Element => {
     const { radius, stroke, fontSize } = CircularProgressSizes[size];
     const { baseCircleClass, progressCircleClass } = color;
-    const innerRadius = radius - stroke * 2;
-    const circumference = innerRadius * 2 * Math.PI;
+    const outerRadius = radius + stroke;
+    const circumference = radius * 2 * Math.PI;
     const progressCircleOffset =
         circumference - (value / target) * circumference;
 
     return (
         <div className="relative flex items-center justify-center">
             <svg
-                height={radius * 2}
-                width={radius * 2}
+                height={outerRadius * 2}
+                width={outerRadius * 2}
                 style={{ transform: 'rotate(90deg)' }}
```

Comment:
Even though it wasn't your code originally but can you please remove magic numbers here